### PR TITLE
pkg/ramfs: add loop devices needed for tcz.

### PR DESCRIPTION
**Pull request by hugelgupf**
Signed-off-by: Christopher Koch <chrisko@google.com>

### CODE REVIEW

File: pkg/ramfs/ramfs.go [hunk 1]
```diff
@@ -45,6 +45,15 @@ var DevCPIO = []cpio.Record{
 	{Info: cpio.Info{Name: "dev/urandom", Mode: c | 0666, Rmajor: 1, Rminor: 9}},
 	{Info: cpio.Info{Name: "etc/resolv.conf", Mode: f | 0644, FileSize: uint64(len(nameserver))}, ReadCloser: cpio.NewBytesReadCloser([]byte(nameserver))},
 	{Info: cpio.Info{Name: "etc/localtime", Mode: f | 0644, FileSize: uint64(len(gmt0))}, ReadCloser: cpio.NewBytesReadCloser([]byte(gmt0))},
+	{Info: cpio.Info{Name: "dev/loop0", Mode: b | 0660, Rmajor: 7, Rminor: 0}},
```
Comment (rminnich) [inline pkg/ramfs/ramfs.go]: I think this is not really needed any more (it used to be) and in any event, it's not near enough devices -- NiChrome uses close to 100. Did you have a problem that this fixed? If so call me surprised.

Note this from d/b/loop.c
         * If max_loop is specified, create that many devices upfront.
         * This also becomes a hard limit. If max_loop is not specified,
         * create CONFIG_BLK_DEV_LOOP_MIN_COUNT loop devices at module
         * init time. Loop devices can be requested on-demand with the
         * /dev/loop-control interface, or be instantiated by accessing
         * a 'dead' device node.

both tcz and losetup in u-root now use loop-control.

Comment (hugelgupf) [reply pkg/ramfs/ramfs.go]: We do at least need /dev/loop-control with this...

tcz was failing with some /dev/loop-control error message for me earlier, but this didn't actually contribute to the fix (I just mknod'ed them, but I don't know if that actually helped?).

Anyway, I'll pay more attention to the error message tomorrow.

**Comment by rminnich** [comment]
I would suggest putting /dev/loop-control in uroot/root.go. 

The cpio is from an earlier time and I'd like to deprecate it as much as we can if that seems OK.

**Comment by rminnich** [comment]
we don't need this any more, right? Can you close it?

**Pull request closed without merging**